<launch>

  <include file="$(find state_manager)/launch/stateManager.launch" />
  <include file="$(find pandora_vision_landoltc)/launch/landoltc_node.launch" />

  <rosparam file="$(find pandora_vision_support)/config/camera_benchmark_params.yaml" 
    command="load"/>

  <include file="$(find pandora_robots_description)/launch/gears/gears_description.launch" />

  <param name="kinect/topic_name" value="/kinect/rgb/image_raw" />
  <param name="publisherMessagePackage" value="sensor_msgs" />
  <param name="publisherMessageType" value="Image"/>
  
  <param name="subscriberTopic" value="/vision/landoltc_alert" />
  <param name="subscriberMessagePackage" value="pandora_vision_msgs" />
  <param name="subscriberMessageType" value="LandoltcAlertsVectorMsg"/>
  
  <param name="algorithm" value="LandoltC"/>
  <param name="dataset_camera" value="kinect"/>
  <param name="dataset_path" value="/test/benchmark/images/images_480p"/>
  
  <test test-name="LandoltcBenchmarkTest" pkg="pandora_vision_landoltc"
    type="landoltc_benchmark_test.py" time-limit="2000.0"/>
  
</launch>
